docs/src/main/asciidoc/kafka-schema-registry-avro.adoc

```diff
@@ -384,7 +420,7 @@ java -Dmp.messaging.connector.smallrye-kafka.apicurio.registry.url=http://localh
 ----
 
 NOTE: By default, the application tries to connect to a Kafka broker listening at `localhost:9092`.
-You can configure the bootstrap server using: `java -Dkafka.bootstrap.servers=... -jar target/quarkus-app/quarkus-run.jar`
+You can configure the bootstrap server using: `java -Dkafka.bootstrap.servers=\... -jar target/quarkus-app/quarkus-run.jar`
```

Comment:
Why did you add a `\`? AFAICS, the current doc is properly rendered on the website.